src/Components/DairyCard.js

```diff
@@ -0,0 +1,84 @@
+import React ,{useState}from 'react'
+import Button from './Button'
+import PropTypes from 'prop-types'
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import Button1 from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+const useStyles = makeStyles({
+    root: {
+      minWidth: 275,
+    },
+    bullet: {
+      display: 'inline-block',
+      margin: '0 2px',
+      transform: 'scale(0.8)',
+    },
+    title: {
+      fontSize: 14,
+    },
+    pos: {
+      marginBottom: 12,
+    },
+  });
+
+function DairyCard({title,subtitle,description}) {
+
+
+    const [readmore,setReadMore] =useState(true);
+    const displaydisciption= readmore? description.slice(0,100):description;
```

Comment:
Wrong variable declaration. This should be like `displayDescription`. And always try to use correct spellings.